### PR TITLE
 omitting the "unused" prompt in code example

### PR DESCRIPTION
**Pull request by shashankshet**
# Omitting the "unused" prompt in code example

updated the doc by removing unused prompt

 
 

----
:books: Documentation preview :books:: https://cpython-devguide--1210.org.readthedocs.build/

### CODE REVIEW

File: documentation/markup.rst [hunk 1]
```diff
@@ -618,7 +618,6 @@ an "unused" primary prompt; this is an example of what *not* to do:
 
    >>> 1 + 1
    2
-   >>>
```
Comment (willingc) [inline documentation/markup.rst:621]: I believe that Lines 614-615 need to be updated as well. 

@ezio-melotti @terryjreedy Please verify that these lines would be better written as:
`After the last line of input or output is presented, there should be no trailing prompt. An example of correct usage is:`

Comment (terryjreedy) [reply documentation/markup.rst:621]: Yes, Good replacement.

**Review by terryjreedy — CHANGES_REQUESTED** [review]

**Comment by shashankshet** [comment]
Update line  614-615 as per review comment, please review

**Review by willingc — APPROVED** [review]
Thanks @shashankshet for the PR and making your first contribution to the dev guide. :snake: :snake: :tada:

**Review by ezio-melotti — APPROVED** [review]

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
@willingc Are you planning to merge this?

**Comment by willingc** [comment]
@terryjreedy Will do! Thanks for reviewing again.

**Pull request merged**